src/Controller/Administrator/Administration/EductionLevelController.php

```diff
@@ -15,8 +15,8 @@ public function listClass(): Response
     }
     public function addClass(string $className, int $educatorId, string $educationLevel): EducationLevel
     {
-        $creator = new CreateObject();
-        $class = $creator->create('educationLevel');
+        $creator = new EducationLevelCreator();
+        $class = $creator->createObject();
```

Comment:
no właśnie - w ten sposób używanie wzorca jest nonsensem bo nic ci to nie przyniosło.